datasource/ethereum/src/block_stream.rs

```diff
@@ -98,6 +104,7 @@ struct BlockStreamContext<S, C, E> {
     subgraph_id: SubgraphDeploymentId,
     reorg_threshold: u64,
     logger: Logger,
+    include_calls_in_blocks: bool,
```

Comment:
Think about making this a static function `include_calls_in_block()` instead of a `bool` property

Reply:
👍 I made this a pure function on the `BlockStreamContext`